internal/integration/integration_test.go

```diff
@@ -80,6 +86,10 @@ func TestIntegration(t *testing.T) {
 		OutputDir:                artifactsOutputDir,
 	}
 
+	if runEmbeddedOmni {
+		runOmni(t)
+	}
```

Comment:
🔥

Reply:
I'll revert that for now. Will come in a separate PR